app/assets/stylesheets/application.scss

```diff
@@ -6,6 +6,7 @@ body {
     background-repeat: no-repeat;
     background-attachment: fixed;
     font-family: Oswald;
+    color: white;
```

Comment:
Se possível usar uma classe ou algo do tipo, para aplicar essa cor só onde for necessário. O Bootstrap já tem a `text-white`, por exemplo.